.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+# Description
+Mandatory, single line description of the pull request, written as though it was an order.
```

Comment:
How does the single line description relate to the title of the PR? Won't it end up being pretty much the same?

Reply:
I think in practice it might. But in that case, we ought to expect a more descriptive pr title, right? A lot of our pr titles wouldn't really fit the bill, I think.

Reply:
Could we split this up as `Summary` where you describe at a high level what the PR does and then the `Description` can contain the details?

Or just remove the single line description and enforce a more descriptive PR title?

Reply:
> Or just remove the single line description and enforce a more descriptive PR title?

I think this is better.

Reply:
19dbe45fc705086e06e7b58ae9da13e3f2e7b98e